.github/CODEOWNERS

```diff
@@ -0,0 +1,31 @@
+# Package code owners
+
+# The listed owners will be automatically added as reviewers for PRs,
+# to ensure code quality and consistency of the package, and identify
+# possible side effects.
+# PRs should still be peer-reviewed by the team opening the PR
+
+# See https://help.github.com/articles/about-codeowners/ for syntax
+
+/pkg/metadata/kubernetes/               @DataDog/burrito
+/pkg/metadata/ecs/                      @DataDog/burrito
+
+/pkg/metrics/percentile/                @DataDog/sandwich
+
+/Dockerfiles/                           @DataDog/kenafeh
+/pkg/tagger/                            @DataDog/kenafeh
+/pkg/util/docker/                       @DataDog/kenafeh @DataDog/burrito
+/pkg/util/ecs/                          @DataDog/kenafeh @DataDog/burrito
+/pkg/util/kubernetes/                   @DataDog/kenafeh @DataDog/burrito
+/pkg/collector/listeners/docker*        @DataDog/kenafeh
+/pkg/collector/corechecks/containers/   @DataDog/kenafeh
+
+/cmd/                                   @DataDog/croissant
+/pkg/aggregator/                        @DataDog/croissant
+/pkg/collector/                         @DataDog/croissant
```

Comment:
Could we make it more fine grained here? autodiscovery, listeners, providers is kenafeh's I think.
If you write
```
/pkg/collector/                         @DataDog/croissant
/pkg/collector/autodiscovery @Datadog/kenafeh
...
```
It should still make croissant the owners of everything in `collector` that is not explicitly listed below it.

Reply:
The matching order is top to bottom, so we need to list the subdirs before. but indeed, I'll add autodiscovery to kenafeh

Reply:
```*       @global-owner1 @global-owner2

# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner```

As I understand it Kenafeh should be after the general `/pkg/collector`. Maybe I'm mis-interpreting?

Reply:
👍  my bad